mobilecoind/clients/python/lib/mobilecoin/client.py

```diff
@@ -142,16 +142,29 @@ def get_monitor_id(self,
     # Utilities
     #
 
-    def generate_entropy(self):
+    def generate_entropy(self) -> bytes:
         """ Generate 32 bytes of entropy using a cryptographically secure RNG.
         """
-        return self.stub.GenerateEntropy(empty_pb2.Empty())
+        response = self.stub.GenerateRootEntropy(empty_pb2.Empty())
+        return response.root_entropy
```

Comment:
This is a breaking change, however I have no idea what uses this code and whether anything will break.
I don't think we currently maintain a changelog or proper versioning for this package.

Reply:
At a glance, all the code in our repo that was calling this has been broken since #779, which renamed the `entropy` member to `root_entropy` (callers still look for `entropy`)

This PR should also adjust those callers:

* http://github.com/jcape/mobilecoin/blob/master/mobilecoind/clients/python/cli/get_public_address.py#L27-L27
* http://github.com/jcape/mobilecoin/blob/master/mobilecoind/clients/python/cli/benchmark.py#L45-L45
* http://github.com/jcape/mobilecoin/blob/master/mobilecoind/clients/python/cli/benchmark.py#L89-L89
* http://github.com/jcape/mobilecoin/blob/master/mobilecoind/clients/python/mailchimp/allocate_mobilecoins.py#L53-L53

Reply:
Is any of this ever going to be used? Maybe we should remove them.

Reply:
I'm fine to accept patches to keep it functioning as long as we have users who haven't switched to [full-service](https://github.com/mobilecoinofficial/full-service) yet.

Reply:
Made changes.

However this comment confuses me, as from what I can see full-service in the other repo is compatible with the python client, and still launches mobilecoind. It uses the same API right? Wouldn't this library still be the approach for third parties to script in Python?

Idk what your guys' priorities are moving forwards, I'm just getting my feet wet with a contribution, and I'm only about halfway through the Mechanics of Mobilecoin. This is an awesome project though, great work!

Reply:
Thank you so much for your contribution @gscrypto :)